frb/galaxies/utils.py

```diff
@@ -15,9 +15,24 @@
 
 from astropy.coordinates import SkyCoord
 import pandas as pd
+import extinction
+from linetools.spectra import xspectrum1d
 
 from frb import frb
 
+def deredden_spec(spectrum:xspectrum1d.XSpectrum1D, ebv:float):
```

Comment:
add a doc string

Reply:
No idea what it does. I think this is related to prospector stuff?

Reply:
I think this is correcting for Galactic extinction but not sure about the input/output data types here.

Reply:
done